### PR TITLE
Remove support for Apiary Blueprint AST

### PR DESCRIPTION
**Pull request by kylef**
We are no longer using Apiary Blueprint AST since https://github.com/apiaryio/apiary/pull/4039 and it's finally time to remove it from Metamorphoses.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,12 +58,6 @@ var mimeType = 'application/vnd.apiblueprint.ast';
 var adapter = metamorphoses.createAdapter(mimeType);
 
 
-// You can also import the adapter directly:
-// -   API Blueprint AST: apiBlueprintAdapter
-// -   Legacy Apiary Blueprint AST: apiaryBlueprintAdapter
-var adapter = metamorphoses.apiBlueprintAdapter;
-
-
```
Comment (pksunkara) [inline README.md:74]: I don't know why this is removed.

**Review by pksunkara — APPROVED** [review]
Just a minor comment.

File: test/serialize-deserialize-test.coffee [hunk 2]
```diff
@@ -1,164 +0,0 @@
...
-    # should get exactly what we need.
-    transformedAst = json
-  )
-  parserAst = ApiaryBlueprintParser.parse(apiaryBlueprint)
-  apiaryBlueprintAdapter.transformAst(parserAst.toJSON())
-  stub.restore()
-  return transformedAst
-
-
-describe('Serialization of the Blueprint interface (Application AST)', ->
```
Comment (kuba-kubula) [inline test/serialize-deserialize-test.coffee:26]: Question: Why was this removed? @kylef @pksunkara 

Should it be rather rewritten to use another parser/adapter, than to simply remove at all?

Comment (pksunkara) [reply test/serialize-deserialize-test.coffee:26]: You are right.

Comment (pksunkara) [reply test/serialize-deserialize-test.coffee:26]: I didn't realise that it was removing the Blueprint Class tests.

Comment (kylef) [reply test/serialize-deserialize-test.coffee:26]: -> https://github.com/apiaryio/metamorphoses/pull/72
